drivers/filesystems/fastfat_new/fatdata.h

```diff
@@ -234,7 +234,7 @@ extern LONG FatDebugTraceIndent;
     }                                                       \
 }
 
-#ifdef _MSC_VER
+#if !defined(__REACTOS__) || defined(_MSC_VER)
```

Comment:
I don't get the idea here. Do we have this warning disabled already?

Reply:
The idea is simply to be explicit about code that does (not) exist upstream.
I don't understand your question. Behavior is unchanged.